packages/browser-destinations/src/destinations/hubspot-web/index.ts

```diff
@@ -44,33 +44,32 @@ export const destination: BrowserDestinationDefinition<Settings, Hubspot> = {
   ],
   settings: {
     portalId: {
-      description: 'The Hub ID of your Hubspot account.',
+      description: 'The Hub ID of your HubSpot account.',
       label: 'Hub ID',
       type: 'string',
       required: true
     },
     enableEuropeanDataCenter: {
-      description: 'Enable the European Data Center.',
-      label: 'Enable European Data Center',
+      description: 'Enable this option if you would like Segment to load the HubSpot SDK for EU data residency.',
+      label: 'Enable the European Data Center SDK.',
       type: 'boolean',
       required: false
     },
     flushIdentifyImmediately: {
-      description: 'Fire a Page View immediately after an Identify to send the data to Hubspot.',
-      label: 'Flush Identify calls immediately',
+      description: 'Enable this option to fire a Page View immediately after an identify call to send the data to HubSpot.',
```

Comment:
Feel free to ignore this suggestion:

```suggestion
      description: 'Enable this option to fire a `trackPageView` HubSpot event immediately after each Segment `identify` call to flush the data to HubSpot immediately.',
```